mmv1/products/networkconnectivity/RegionalEndpoint.yaml

```diff
@@ -0,0 +1,127 @@
+# Copyright 2024 Google Inc.
+# # Licensed under the Apache License, Version 2.0 (the "License");
+# # you may not use this file except in compliance with the License.
+# # You may obtain a copy of the License at
+# #
+# #     http://www.apache.org/licenses/LICENSE-2.0
+# #
+# # Unless required by applicable law or agreed to in writing, software
+# # distributed under the License is distributed on an "AS IS" BASIS,
+# # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# # See the License for the specific language governing permissions and
+# # limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'RegionalEndpoint'
+description: |
+  Regional Private Service Connect (PSC) endpoint resource.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Access regional Google APIs through endpoints': 'https://cloud.google.com/vpc/docs/access-regional-google-apis-endpoints'
+  api: 'https://cloud.google.com/network-connectivity/docs/reference/networkconnectivity/rest/v1/projects.locations.regionalEndpoints'
+base_url: 'projects/{{project}}/locations/{{location}}/regionalEndpoints'
+self_link: 'projects/{{project}}/locations/{{location}}/regionalEndpoints/{{name}}'
+immutable: true
+timeouts: !ruby/object:Api::Timeouts
+  insert_minutes: 20
+  delete_minutes: 20
+create_url: 'projects/{{project}}/locations/{{location}}/regionalEndpoints?regional_endpoint_id={{name}}'
+delete_url: 'projects/{{project}}/locations/{{location}}/regionalEndpoints/{{name}}'
+autogen_async: true
+async: !ruby/object:Api::OpAsync
+  actions: ['create', 'delete']
+  operation: !ruby/object:Api::OpAsync::Operation
+    base_url: '{{op_id}}'
+  result: !ruby/object:Api::OpAsync::Result
+    path: 'response'
+  status: !ruby/object:Api::OpAsync::Status
+    path: 'done'
+    complete: true
+    allowed:
+      - true
+      - false
+  error: !ruby/object:Api::OpAsync::Error
+    path: 'error'
+    message: 'message'
+
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'network_connectivity_regional_endpoint_regional_access'
+    primary_resource_id: 'default'
+    vars:
+      rep_name: 'my-rep'
+      rep_target_google_api: 'spanner.me-central2.p.rep.googleapis.com'
+      rep_access_type: 'REGIONAL'
+      rep_address: 'my-address'
+      rep_network: 'my-network'
+      rep_subnetwork: 'my-subnetwork'
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'network_connectivity_regional_endpoint_global_access'
+    primary_resource_id: 'default'
+    vars:
+      rep_name: 'my-rep'
+      rep_target_google_api: 'spanner.me-central2.p.rep.googleapis.com'
+      rep_access_type: 'GLOBAL'
+      rep_address: 'my-address'
+      rep_network: 'my-network'
+      rep_subnetwork: 'my-subnetwork'
+
+parameters:
+  - !ruby/object:Api::Type::String
+    name: 'name'
+    output: true
```

Comment:
`output: true` is not needed, as `name` is provided by the user.

Reply:
In the proto definition it's actually an OUTPUT_ONLY field, so I removed it from tests too.

Reply:
As the parameter, `name` is actually `regionalEndpointId`, and it is not OUTPUT_ONLY field and should be provided by the user.

Reply:
Makes sense, thank you!